libcusp/include/galois/graphs/MiningPartitioner.h

```diff
@@ -1131,7 +1132,7 @@ class MiningGraph : public DistGraph<NodeTy, EdgeTy> {
         uint64_t cur = *graph.edge_begin(lsrc, galois::MethodFlag::UNPROTECTED);
         uint64_t cur_end = *graph.edge_end(lsrc);
         assert((cur_end - cur) == gdst_vec.size());
-        deserializeEdges(graph, rb, gdst_vec, cur, cur_end);
+        deserializeEdges(graph, gdst_vec, cur, cur_end);
```

Comment:
I'm assuming this change is correct; variable changed completely.

Reply:
It's an unused function argument - the change is required to fix the build.